### PR TITLE
docs: Add pointer to `debugPrintReports` for AER

### PR DESCRIPTION
**Pull request by zionts**
This allows users to validate why metrics may not be working if they encounter errors.

### CODE REVIEW

File: docs/source/features/metrics.md [hunk 1]
```diff
@@ -36,6 +36,21 @@ The API key can also be set with the `ENGINE_API_KEY` environment variable. Sett
 ENGINE_API_KEY=YOUR_API_KEY node start-server.js
 ```
 
+Not sure if metrics reporting is working? You can set the [`debugPrintReports` option](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/agent.ts#L429-L433) in the `engine` option of the Apollo Server constructor to see a message indicating the success of transmission like so:
```
[the remark this answers is not in the document]
Comment (zionts) [reply docs/source/features/metrics.md]: Love it!

File: docs/source/features/metrics.md [hunk 2]
```diff
@@ -36,6 +36,21 @@ The API key can also be set with the `ENGINE_API_KEY` environment variable. Sett
...
+
+```js{9}
+const { ApolloServer } = require("apollo-server");
+
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+  engine: {
+    apiKey: "YOUR API KEY HERE",
```
[the remark this answers is not in the document]
Comment (zionts) [reply docs/source/features/metrics.md]: sgtm

File: docs/source/features/metrics.md [hunk 3]
```diff
@@ -57,6 +57,22 @@ You can set environment variable values on the command line as seen below, or wi
 ENGINE_API_KEY=YOUR_API_KEY ENGINE_SCHEMA_TAG=development node start-server.js
 ```
 
+### Debugging
```
Comment (StephenBarlow) [inline docs/source/features/metrics.md]: Expand to "Debugging Graph Manager reporting

File: docs/source/features/metrics.md [hunk 4]
```diff
@@ -57,6 +57,22 @@ You can set environment variable values on the command line as seen below, or wi
...
+
+You can set the [`debugPrintReports` option](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/agent.ts#L429-L433) in the `engine` section of the Apollo Server constructor to see debug output regarding trace transmission.  For example:
```
Comment (StephenBarlow) [inline docs/source/features/metrics.md]: Apollo Server -> `ApolloServer`

Comment (StephenBarlow) [reply docs/source/features/metrics.md]: "see debug output regarding trace transmission" -> "automatically log debugging information for all reporting requests sent to Graph Manager"

Comment (abernix) [reply docs/source/features/metrics.md]: Resolved via 8b0f8136.

File: docs/source/features/metrics.md (hunk 3 above)
Comment (abernix) [reply docs/source/features/metrics.md]: Resolved via 10dc7c1d.

**Pull request merged**